addons/udes_stock/tests/common.py

```diff
@@ -410,12 +410,24 @@ def create_move(cls, pickings, products_info, **kwargs):
     @classmethod
     def create_move_line(cls, moves, qty, **kwargs):
```

Comment:
Nit: Should probably be move not moves now

Reply:
Yep, good spot!

Reply:
Have now renamed this. :)